src/transform.ts

```diff
@@ -134,6 +137,21 @@ module.exports = exportedTokens;`,
   };
 }
 
+function processPostCSS(src: string, filename: string): TransformedSource {
+  return {code: `
+const postcss = require("postcss");
+const cssSrc = ${JSON.stringify(src)};
+postcss([require("tailwindcss")(), require("autoprefixer")()])
+  .process(cssSrc, { from: ${JSON.stringify(filename)} })
+  .then((result) => {
+    const style = document.createElement("style");
+    style.type = "text/css";
+    style.appendChild(document.createTextNode(result.css.replace(/\\\\/g, '')));
```

Comment:
This part `result.css.replace(/\\\\/g, '')` is hard to read even more regex is already complex.

It is parsed as `result.css.replace(/\\/g, ''))` on runtime.